src/sql-parser/src/ast/defs/name.rs

```diff
@@ -333,6 +338,9 @@ pub enum IdentError {
         suffix: String,
         attempts: usize,
     },
+
+    #[error("invalid identifier: '{}'", .0)]
+    InvalidName(String),
```

Comment:
I think a more descriptive name here would be nice!

```suggestion
    WouldCausePathTraversalInUrl(String),
```